client/my-sites/plans/current-plan/ecommerce-trial/index.jsx

```diff
@@ -0,0 +1,242 @@
+	// https://github.com/moment/moment/issues/3341
+	const readableExpirationDate = eCommerceTrialExpiration?.toDate().toLocaleDateString( locale, {
+		month: 'long',
+		day: 'numeric',
+	} );
+
+	return (
+		<>
+			<BodySectionCssClass bodyClass={ [ 'is-trial-plan' ] } />
+
+			<Card className="e-commerce-trial-current-plan__trial-card">
+				<div className="e-commerce-trial-current-plan__trial-card-content">
+					<p className="e-commerce-trial-current-plan__card-title">
+						{ translate( 'You’re in a free trial' ) }
+					</p>
+					<p className="e-commerce-trial-current-plan__card-subtitle">
+						{
+							// Still need to populate the date correctly
+							translate(
+								'Your free trial will end in %(daysLeft)d day. Sign up to a plan by %(expirationdate)s to unlock new features and keep your store running.',
+								'Your free trial will end in %(daysLeft)d days. Sign up to a plan by %(expirationdate)s to unlock new features and keep your store running.',
+								{
+									count: eCommerceTrialDaysLeft,
+									args: {
+										daysLeft: eCommerceTrialDaysLeft,
+										expirationdate: readableExpirationDate,
+									},
+								}
+							)
+						}
+					</p>
+					<Button className="e-commerce-trial-current-plan__trial-card-cta" primary>
+						{ translate( 'Get Commerce' ) }
+					</Button>
+				</div>
+				<div className="plans__chart-wrapper">
+					<DoughnutChart progress={ trialProgress } text={ eCommerceTrialDaysLeft } />
+					<br />
+					<span className="plans__chart-label">
+						{ isTrialExpired
+							? translate( 'Your free trial has expired' )
+							: translate( 'day left in trial', 'days left in trial', {
+									count: eCommerceTrialDaysLeft,
+							  } ) }
+					</span>
+				</div>
+			</Card>
+
+			<h2 className="e-commerce-trial-current-plan__section-title">
+				{ translate( 'What’s included in your free trial' ) }
+			</h2>
+			<div className="e-commerce-trial-current-plan__included-wrapper">
+				{ whatsIncluded.map( ( feature ) => (
+					<FeatureIncludedCard
+						key={ feature.title }
+						illustration={ feature.illustration }
+						title={ feature.title }
+						text={ feature.text }
+						showButton={ feature.showButton }
+						buttonText={ feature.buttonText }
+					></FeatureIncludedCard>
+				) ) }
+
+				{ ! displayAllIncluded ? (
+					<Button
+						className="e-commerce-trial-current-plan__included-view-all"
+						onClick={ viewAllIncludedFeatures }
+					>
+						{ translate( 'View all' ) }
+					</Button>
+				) : null }
+			</div>
+
+			<h2 className="e-commerce-trial-current-plan__section-title">
+				{ translate( 'Do you want more?' ) }
+			</h2>
+			<p className="e-commerce-trial-current-plan__section-subtitle">
+				{ translate( 'The free trial doesn’t support the following features.' ) }
+				<br />
+				{ translate( 'Get the most value out of WooCommerce and get Pro.' ) }
```

Comment:
This wording feels like it's tied to our branding. Maybe switch it to "upgrade" instead of "get Pro"?
```suggestion
				{ translate( 'Get the most value out of WooCommerce and upgrade.' ) }
```